ci/jenkins/test.sh

```diff
@@ -277,6 +277,15 @@ function wait_for_antrea_windows_pods_ready {
         done
         sleep 10
     done
+    echo "===== Check Interface DHCP Status ====="
```

Comment:
Create a general function if the check is same for antrea-agent Pod and process cases?

Reply:
Yes, added this function.